modules/lite/src/deckgl.js

```diff
@@ -116,7 +110,7 @@ export default class DeckGL extends Deck {
   }
 
   setProps(props) {
-    if (props.onViewportChange !== this._onViewportChange) {
+    if (this._initialized && props.onViewportChange !== this._onViewportChange) {
```

Comment:
Because we cannot bind `this._onViewportChange` before calling `super(props)`, `props.onViewportChange` cannot be properly populated the first time round.
This can be easily avoided if we upgrade to stage-2.

Reply:
* I'm curious, how can this be avoided in stage-2? Isn't that just different syntax, bound by the same limitations? 
* Regardless, I don't think it is worth breaking the ability to run untranspiled in Node and browser for this. 
* I suppose this particular module could opt in to stage-2, but not worth the confusion IMHO.

I am trying to put up a PR for default `onViewStateChange` in `Deck` maybe that would make the issue moot.

Reply:
Using class properties, a.k.a.,
```
this._onViewportChange = () => {};
```
The method is automatically bound.

Reply:
Yes, though the interesting question is, when does it happen? Before or after the `super()` call? Is this construct actually allowed to break the "don't assign to `this` before `super()` is called" rule? And especially if it is being transpiled, how can the transpiled code break the rule?